NuGet.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="TransportFeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
```

Comment:
nit: We have been trying to name this the BlobFeed as opposed to Transport feed.

Reply:
I will rename it...